sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -176,8 +174,12 @@ async def read(
         :returns: Dict representing the retrieved container.
         :rtype: Dict[str, Any]
         """
+        session_token = kwargs.get('session_token')
         if session_token is not None:
-            kwargs['session_token'] = session_token
+            warnings.warn(
+                "The 'session_token' flag does not apply to this method and will be removed in the future.",
```

Comment:
how about updating the message like this? and for others as well. Letting customers know that it is always ignored may help with any concerns they may have about deprecation?
```suggestion
                "The 'session_token' flag does not apply to this method and is always ignored even if passed. It will now be removed in the future.",
```

Reply:
sounds good, I'll go ahead and do that